insights/core/plugins.py

```diff
@@ -96,6 +124,12 @@ def invoke(self, broker):
             log.debug(cpe)
             broker.add_exception(self.component, cpe, traceback.format_exc())
```

Comment:
Better apply the `inner` and `recursive`ly `_get_spec_name` to each of the 3 `*Exception`s here in the invoke to avoid generating `meta_data` for the local specs that do not need to be dehydrated.

Reply:
Updated adding to each exception, also removed the persist addition.